addons/payment_odoo/views/payment_odoo_menus.xml

```diff
@@ -1,16 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
-
     <menuitem id="root_adyen_menu"
         name="Odoo Payments"
         parent="account.menu_finance_configuration"
-        sequence="4">
+        sequence="3">
         <menuitem id="menu_adyen_transaction"
             action="odoo_payments.adyen_transaction_action"/>
         <menuitem id="menu_adyen_balance"
             action="odoo_payments.adyen_balance_action"/>
         <menuitem id="menu_adyen_account"
             action="odoo_payments.adyen_account_action"/>
     </menuitem>
-
```

Comment:
Why do you remove the empty lines around the menuitem node ?

Reply:
Are they useful ? :P
Now, I even moved each menuitem on one line as it does not exceed 120 characters per line. For me, it's more readable, but if it's not ok, I can change of course ;)

Reply:
Not really, that's the way we tend to do things in payment team, but like for most guidelines, everybody has its own point of view :D.
I'd say that in this case, the base code is clean, so we should just avoid "useless" diff ;).